Tasks/AzureFunctionOnKubernetesV1/task.loc.json

```
@@ -0,0 +1,192 @@
+{
+  "id": "FD1AA5B9-400C-4F4B-9C0B-069BA74E53C6",
+  "name": "AzureFunctionOnKubernetes",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpUrl": "https://aka.ms/func-k8s",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Deploy",
+  "preview": true,
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 216,
```

The minor version in the main task.json is 218, but here the version is 216.
Should we update it too?

```suggestion
    "Minor": 218,
```